test/merchant_test.rb

```diff
@@ -50,10 +52,46 @@ def test_favorite_customer_returns_customer_with_most_successful_transactions
     assert_equal "Joey", result.first_name
   end
 
+  def test_customers_with_pending_invoices_returns_collection_of_customers_with_unpaid_invoices
+    load_data
+    result = @se.merchant_repository.merchants[2].customers_with_pending_invoices
+    assert_equal 0, result.count
+  end
+
   private
 
   def fixtures_directory
     File.expand_path('../../data/fixtures', __FILE__)
   end
 
+  def load_data
+    repo_data = {
+      merchants: [
+       { id: 10, name: "merchant_1" },
+       { id: 20, name: "merchant_2" },
+       { id: 30, name: "merchant_3" }
+     ],
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.